### DISCUSSION TITLE
guide/drivers/aliyundrive_open

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#197] [Announcements]
# guide/drivers/aliyundrive_open

Aliyundrive Open, using the official authorization API development. import { ref } from 

https://alist.nn.ci/guide/drivers/aliyundrive_open.html

**Comment by DamienMM** [comment]
这破阿里云盘真是受够了，今天挂不上名天又挂比不上，天天有问题天天不一样，人家百度网盘和夸克网盘什么问题都没出过稳得一批，这垃圾玩意以后说啥也不能给他充钱了。

**Reply by LittleJake** [reply]
国内网盘生态太封闭了，国外网盘都有官方api。

**Comment by nbgwh1234** [comment]
failed get objs: failed to list objs: invalid character '<' looking for beginning of value

**Reply by licher-liu** [reply]
Same error, need a solution to address this issue. Help...

**Comment by VincentYD** [comment]
Failed init storage: failed to refresh token: refresh token is empty, resp: 404 page not found,  这个是什么原因啊 整吐了呀

**Reply by VincentYD** [reply]
有没有大佬 帮帮我

**Comment by vcxe** [comment]
阿里云现在限流4mb左右，高清电影看着费劲了卡的不要不要的

**Comment by vvwn** [comment]
用Alist挂载阿里网盘到本地盘  普通用户 和会员在传输速度上有区别吗
